source/blog/2020-10-27-release-notes.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: October 2020 Release Notes - Version 20.10.0 🎃
+published: true
+post_author:
+  display_name: Jing Yan
+  twitter: jyan
```

Comment:
Replace with 'freshlemonfish'. 😃

Reply:
updated, for a second I thought it was a joke 😅